interface/super/edit_globals.php

```diff
@@ -603,12 +610,12 @@ function validate_file(){
     }
     ++$i;
    }
-    if(trim(strtolower($fldid)) == 'portal_offsite_address_patient_link' && $GLOBALS['portal_offsite_enable'] && $GLOBALS['portal_offsite_providerid']){
+    if(trim(strtolower($fldid)) == 'portal_offsite_address_patient_link' && array_key_exists('portal_offsite_enable', $GLOBALS) && $GLOBALS['portal_offsite_enable'] && array_key_exists('portal_offsite_providerid', $GLOBALS) && $GLOBALS['portal_offsite_providerid']){
```

Comment:
The first two for perhapse, as **"1"** or **"0"** are used (I think), and the third only if **"0"** is never a valid ID.